dashboard/app/views/layouts/application.html.haml

```diff
@@ -36,6 +35,16 @@
     - if current_user && current_user.age.nil?
       = render partial: 'layouts/age_interstitial'
 
+    #gdpr-dialog
+    - gdpr_data = {}
+    - if current_user
+      - already_accepted = !current_user.data_transfer_agreement_accepted.nil?
```

Comment:
While this should work under current implementation, I wonder if this should explicitly check for `true`?

Reply:
Agreed - let's assume we might end up with `false` in that field at some point in the future, and let the user model validation be the one place we assume/enforce it can only be true.

I have a separate issue with that validation as it exists right now, but we'll talk offline today. :grin: